specs/error_state/ErrorCodeStore.md

```diff
@@ -16,9 +14,7 @@ code_store_cost = CODE_DEPOSIT_BYTE_COST * len(bytecodes)
 - If `code_store_cost` > gas left, it is `CodeStoreOutOfGas` case.
 - If returned bytecode length > `MAXCODESIZE`, it is `MaxCodeSizeExceeded` case.  
 
-In circuit bus mapping side, check these two code store errors in [here](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/bus-mapping/src/circuit_input_builder/input_state_ref.rs#L1148&L1155). When executing opcode is `RETURN` and call context is creating(`call.is_create == true`) meanwhile.  
-
-Even though errors occur in `CREATE` kind opcodes, it is special not checking error in executing opcode `CREATE` directly. Circuit implementation takes similar strategy, not constrain error directly in CREATE opcodes, but in `RETURN` step context. Following this way it easy to get the key property state `length` and construct constraints against it.
+In circuit bus mapping side, check these two code store errors in [here](https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/8a633f7c3de2da72f0817def57c1703241cced97/bus-mapping/src/circuit_input_builder/input_state_ref.rs#L1296-L1304). This error happens only when current opcode is `RETURN` and it's a `CREATE`/`CREATE2` call (`call.is_create == true`). We can't get contract bytecode length in `CREATE`/`CREATE2` opcodes and it's only available in `RETURN` opcode so we handle these two errors in `RETURN` opcode.
```

Comment:
Nitpick: perhaps "On the circuit bus mapping side, the checks for these two code store errors are [here]"?

Reply:
fixed in [d630df6](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/475/commits/d630df687049ed065be459186951093609f74b15)